<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>K6GTE AuGratin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(45,45,45);
color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>16</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="activator_call">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Activator Call</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_designator">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>XX-####</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="comments">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>451</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Comments</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rst_sent">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>599</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rst_recieved">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>270</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>599</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_field">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>HHMMSS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_field">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>YYYYMMDD</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="webEngineView">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>411</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(66, 66, 66);
</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_band">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>160</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_mode">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-FT*</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FT8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FT4</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="activator_name">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Activator Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_name">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Park Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_section">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Section</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_state">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_grid">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>190</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Grid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_distance">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="park_direction">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mode_field">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="band_field">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>000M</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="freq_field">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string>000000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mycall_field">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mygrid_field">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="log_button">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>310</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,128,0);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Log It</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Km</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Deg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>270</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>RST TX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>270</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>RST RX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Your Grid:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Your Call:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
